src/config/basic_config.py

```diff
@@ -4,13 +4,20 @@
 note       : 相关测试配置项
 create_time: 2020/4/22 5:17 下午
 """
-code_path = '/home/runner/work/hugegraph-test/hugegraph-test/graph'
+import os.path
+
+# code_path = '/home/runner/work/hugegraph-test/hugegraph-test/graph'
+code_path = os.path.dirname(os.path.realpath(__file__)) + "/../../graph"
 
 is_auth = False
 is_https = False
 
 # server
-server_git = {'branch': 'c0dff5b233e853716ca6f3f28a5cda05e6f3d639', 'url': 'https://github.com/hugegraph/hugegraph.git'}
+server_git = {
+    # 'branch': 'master',
+    'branch': '20d1e5228e39bd974079891fc713574fb14798c5',
+    'url': 'https://github.com/apache/incubator-hugegraph.git'
```

Comment:
```suggestion
    'url': 'https://github.com/apache/hugegraph.git'
```
I think we can avoid name changed in future in the way